rivus/utils/prerun.py

```diff
@@ -0,0 +1,45 @@
+"""
+    Collection of small rivus related util functions
+    
+    In use to avoid multiple solutions of the same
+    problem.
+"""
+from multiprocessing import cpu_count
+
+def setup_solver(optim, logfile='solver.log', guroTimeLimit=12000, guroMIPFocus= 2, guroMIPGap=.001, guroThreads=None):
```

Comment:
Please wrap this line to the PEP8-endorsed length. Also: there is a space after `guroMIPFocus=`.